BUILD

```diff
@@ -0,0 +1,60 @@
+# Description:
+# A Motion Animator creates performant, interruptible animations from motion specs.
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+load("@bazel_ios_warnings//:strict_warnings_objc_library.bzl", "strict_warnings_objc_library")
+
+strict_warnings_objc_library(
+    name = "MotionAnimator",
+    srcs = glob([
+        "src/*.m",
+        "src/private/*.m",
+    ]),
+    hdrs = glob([
+        "src/*.h",
+        "src/private/*.h",
+    ]),
+    deps = [
+      "@motion_interchange_objc//:MotionInterchange"
+    ],
+    enable_modules = 1,
+    includes = ["src"],
+    visibility = ["//visibility:public"],
+)
+
+load("@build_bazel_rules_apple//apple:swift.bzl", "swift_library")
```

Comment:
Nit/Style: Please place all loads at the top of the file.

Reply:
Done.